web/pandas/pdeps/0001-purpose-and-guidelines.md

```diff
@@ -12,15 +12,21 @@ A PDEP (pandas enhancement proposal) is a proposal for a **major** change in
 pandas, in a similar way as a Python [PEP](https://peps.python.org/pep-0001/)
 or a NumPy [NEP](https://numpy.org/neps/nep-0000.html).
 
-Bug fixes and conceptually minor changes (e.g. adding a parameter to a function)
-are out of the scope of PDEPs. A PDEP should be used for changes that are not
-immediate and not obvious, and are expected to require a significant amount of
-discussion and require detailed documentation before being implemented.
-
-PDEP are appropriate for user facing changes, internal changes and organizational
-discussions. Examples of topics worth a PDEP could include moving a module from
-pandas to a separate repository, a refactoring of the pandas block manager or
-a proposal of a new code of conduct.
+Bug fixes and conceptually minor changes (e.g. adding a parameter to a function) are out of the
+scope of PDEPs. A PDEP should be used for changes that are not immediate and not obvious, when
+everybody in the pandas community needs to be aware of the possibility of an upcoming change.
+Such changes require detailed documentation before being implemented and frequently lead to a
+significant discussion within the community.
+
+PDEPs are appropriate for user facing changes, internal changes and significant discussions.
+Examples of topics worth a PDEP could include substantial API changes, breaking behavior changes,
+moving a module from pandas to a separate repository, a refactoring of the pandas block manager
+or a proposal of a new code of conduct. It is not always trivial to know which issue has enough
```

Comment:
Personally, I find the "proposal for a new code of conduct" the odd one in this list. I would say that's more a topic related to community organization, for which we don't necessarily plan to use the PDEP process?